src/universal/conf/ethereumHF.conf

```diff
@@ -26,6 +26,8 @@ mantis {
     # Setting both values to zero disables difficulty bomb delay
     difficulty-bomb-pause-block-number = "0"
     difficulty-bomb-continue-block-number = "0"
+    # Disabled for ethereumHF
+    difficulty-bomb-removal-block-number = "1000000000000000000"
```

Comment:
Minor comment (no changes required). I still prefer an `Option` here, but I remember it was already discussed for `eip161-block-number`